### ISSUE TITLE
Add tagged Docker image

### ISSUE THREAD

**Issue by lithammer** [danihodovic/celery-exporter#35]
It would be nice with a version tagged Docker image instead of just `latest`. Don't want any surprises when deploying in production.

**Comment by danihodovic** [comment]
Good idea, I'll see if I can combine git tags and automated Docker hub builds in the following weeks.

**Comment by danihodovic** [comment]
I've manually started pushing tagged images (0.0.2, 0.0.3), but I'm yet to add an automated process.

**Comment by itskingori** [comment]
> Good idea, I'll see if I can combine git tags and automated Docker hub builds in the following weeks.

I've done this in https://github.com/danihodovic/celery-exporter/pull/329 where it will configure publish `<tag>` and `latest` on release to both GitHub Container Registry and Docker Hub.
